pkg/helm/helm.go

```diff
@@ -7,13 +7,26 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strings"
 
 	"github.com/ghodss/yaml"
 	"github.com/segmentio/kubeapply/pkg/util"
 	log "github.com/sirupsen/logrus"
 )
 
+var (
+	sep = regexp.MustCompile("(?:^|\\s*\n)---\\s*")
```

Comment:
Maybe we want to match the YAML spec a bit more closely? I'm not seeing anything saying spaces are allowed at the beginning of the line, but a value may be specified right after: https://yaml.org/spec/1.2/spec.html#id2760395

Reply:
Eh, I think we should just use a YAML library that supports handling the splits directly (unfortunately, `ghodss` does not at the moment). Added a TODO for this.